charts/coralogix-operator/templates/coralogix_operator_rules.yaml

```
@@ -0,0 +1,25 @@
+apiVersion: coralogix.com/v1alpha1
+kind: RecordingRuleGroupSet
+metadata:
+  name: {{ include "coralogixOperator.fullname" . }}
+spec:
+  groups:
+    - name: reconcile_error_rates
+      intervalSeconds: 60
+      rules:
+        - expr: sum(rate(controller_runtime_reconcile_total{pod=~".*coralogix-operator.*"}[2m])) by (controller)
+          record: coralogix_operator:reconcile_total:controller
+        - expr: sum(rate(controller_runtime_reconcile_errors_total{pod=~".*coralogix-operator.*"}[2m])) by (pod)
+          record: coralogix_operator:reconcile_errors_total:pod
+    - name: unsynced_resources
+      intervalSeconds: 60
+      rules:
+        - expr: sum(cx_operator_resource_info{status=~"RemoteUnsynced"}) by (kind)
+          record: coralogix_operator:unsynced_resources_total:kind
```

```suggestion
          record: coralogix_operator:unsynced_resources_count:kind
```
`total` is for counters, so it may mislead users to think this rr is accumulative.
You can also use this rr in the alert, no?